helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -418,16 +418,9 @@ internal fun String.wrapAt(column: Int): String {
 
     var text = this
     while (text.isNotEmpty()) {
-        val spaceIndex = text.indexOf(' ', column)
-
-        var index = spaceIndex
-        while (index > column) {
-            // Find the last space before the column.
-            while (index > 0 && text[--index] != ' ');
-        }
-
-        // If there is no space before the column, take the first space after the column.
-        if (index == 0) index = spaceIndex
+        val firstSpaceAfterColumnIndex = text.indexOf(' ', column)
+        val lastSpaceBeforeColumnIndex = text.lastIndexOf(' ', column - 1)
+        val index = lastSpaceBeforeColumnIndex.takeUnless { it == -1 } ?: firstSpaceAfterColumnIndex
```

Comment:
Nit: Do you think it makes sense to inline `firstSpaceAfterColumnIndex` now so it's only calculated in the of the fallback?

Reply:
I tried out the inlineing when writing this code, decided against it as I found it less readable.